README.md

```diff
@@ -26,7 +26,7 @@ Submitted pull requests should be made up of commits with messages that are suit
 
 The outputs of the build are
 
-- A system image, ``output/images/ot2-system.zip``, good for doing a software update
+- A system image, ``output/images/ot2-system.zip``, good for doing a software update. (If your build is unsigned you will need to disable signature checking in the robot's update server config (``/var/lib/otupdate/config.json``) and restart.)
```

Comment:
you can actually just restart the component that handles updates, the [update server](https://github.com/Opentrons/opentrons/tree/edge/update-server): `systemctl restart opentrons-update-server`

Reply:
Thanks -- done